<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Студенты и Диаграммы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="loadDataButton">
        <property name="text">
         <string>Загрузить данные</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buildChartsButton">
        <property name="text">
         <string>Обновить диаграмму</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addDataButton">
        <property name="text">
         <string>Добавить данные</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteDataButton">
        <property name="text">
         <string>Удалить данные</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveDataButton">
        <property name="text">
         <string>Сохранить данные</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="chartsLayout">
      <item>
       <layout class="QVBoxLayout" name="chartLayout1"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="chartLayout2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
